tests/tensor/rewriting/test_basic.py

```diff
@@ -1300,6 +1302,20 @@ def test_local_join_make_vector():
     assert check_stack_trace(f, ops_to_check="all")
 
 
+def test_local_sum_make_vector():
+    a, b, c = scalars("abc")
+    mv = MakeVector(config.floatX)
+    output = mv(a, b, c).sum()
+
+    func = function([a, b, c], output)
```

Comment:
No need to compile a function if you're not evaluating it. You can call `rewrite_graph` from `graph.utils`.